src/roman-numerals/test.js

```diff
@@ -37,8 +37,10 @@ describe("Roman to Numerals solution", () => {
   });
 
   test("Deductions with X", () => {
-    expect(romanToDecimal("XL")).toBe(40);
-    expect(romanToDecimal("XC")).toBe(90);
+    expect(romanToDecimal("XL")).toBe(49);
+    expect(romanToDecimal("XC")).toBe(99);
+    expect(romanToDecimal("XD")).toBe(499);
+    expect(romanToDecimal("XM")).toBe(999);
```

Comment:
These files should not be modified. The one on upstream is right as you can see here.
Remove this change as its making your tests fail.